convert2rhel/unit_tests/grub_test.py

```diff
@@ -422,9 +447,9 @@ def test_efibootinfo(monkeypatch, is_efi, subproc_called, subproc, exception):
             grub.EFIBootInfo()
     else:
         efibootinfo_obj = grub.EFIBootInfo()
-        assert len(efibootinfo_obj.entries) == 4
-        assert efibootinfo_obj.current_bootnum == "0004"
-        assert efibootinfo_obj.boot_order == ("0004", "0002", "0000", "0003")
+        assert len(efibootinfo_obj.entries) == total_entries
+        assert efibootinfo_obj.current_bootnum == current_bootnum
+        assert efibootinfo_obj.boot_order == boot_order
```

Comment:
> @SpyTec I've changed the parameters for the test a bit, this way we can pass different values for the parametrization

Extended parameterize looks good! Nothing in this test asserts that the RegEx works as expected though. If you use the old RegEx you'll notice that the tests give a false positive here


One more assertion can be added
```suggestion
        assert len(efibootinfo_obj.entries) == total_entries
        assert efibootinfo_obj.current_bootnum == current_bootnum
        assert efibootinfo_obj.boot_order == boot_order
        assert current_bootnum in efibootinfo_obj.entries
```

Reply:
Done!